cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -134,8 +134,12 @@ func (p *Provisioner) GetNodeObjectFromHostName(hostName string) (*v1.Node, erro
 		Limit:         1,
 	}
 	nodeList, err := p.kubeClient.CoreV1().Nodes().List(listOptions)
-	if err != nil {
-		return nil, errors.Errorf("Unable to get the Node with the NodeHostName")
+	if err != nil || nodeList.Items == nil || len(nodeList.Items) == 0 {
```

Comment:
Do we need both the checks, for `Items` not nil and `len(Items)`

Reply:
@AmitKumarDas -- what is the idiomatic go way for above checks?

Reply:
Len check only for arrays is good